src/ExperimentTable.js

```diff
@@ -60,6 +60,15 @@ class ExperimentTable extends React.Component {
       )
   }
 
+  _isContains(data, value) {
+    let contains = false
+    Object.keys(data).some(key => {
```

Comment:
Room for simplification here:
```javascript
_isContains(data, value) {
  return Object.keys(data).some(key => typeof ...)
}
```

Maybe the logic itself can be simplified, but we can get to that after the “return noise” and throw-away variables are removed.

Reply:
Done